{
  "Header": {
    "Original": "CVE-2025-55554 (pytorch v2.8.0 was discovered to contain an integer overflow in the co ...)",
    "ID": "CVE-2025-55554",
    "Description": "(pytorch v2.8.0 was discovered to contain an integer overflow in the co ...)"
  },
  "Annotations": [
    {
      "Original": "- pytorch \u003cunfixed\u003e (bug #1116534)",
      "Type": "package",
      "Package": "pytorch",
      "Kind": "unfixed",
      "BugNo": 1116534
    },
    {
      "Original": "NOTE: https://github.com/pytorch/pytorch/issues/151510",
      "Type": "NOTE",
      "Description": "https://github.com/pytorch/pytorch/issues/151510"
    }
  ]
}